app/models/requests/patron.rb

```diff
@@ -2,6 +2,11 @@
 
 module Requests
   class Patron
+    class ServerError < StandardError; end
```

Comment:
@jrgriffiniii Since we no longer reference these errors in the code can these error class declarations be removed?